create_db.sh

```diff
@@ -11,16 +11,27 @@ if [[ $(docker volume ls -f name=wordle_dbdata) == "wordle_dbdata" ]]; then dock
 # Remove the volume
 docker volume rm wordle_dbdata
 
-# Runs the POSTGRES container
+# Run the POSTGRES container
 echo "Now running the postgres container"
 container_id=$(docker run -it -d  -e POSTGRES_USER=${pg_user} -e POSTGRES_DB=${pg_db} -e POSTGRES_PASSWORD=${pg_pass} -v $PWD:/opt/sql -v wordle_dbdata:/var/lib/postgresql/data postgres:13) 
 
-while ! docker exec ${container_id} pg_isready; do
+while true; do
+  docker exec ${container_id} pg_isready
+  if [[ $? -eq 0 ]]; then
+    # The server is ready, so exit the loop
+    break
+  fi
   # Print a message and sleep for 3 seconds before checking again
-  echo "Waiting for the database to start"
-  sleep 1
+  echo "Waiting for the database to start..."
+  sleep 3
 done
 
+# Check the exit status of the docker exec command
+if [[ $? -ne 0 ]]; then
```

Comment:
Can this even happen? I think if you reached this point it's because `$?` is 0. Anyway, no harm done.
Food for thoughts: the reason why Docker Compose and health checks are great is because they simplify all this work. For example, your code is waiting forever, so a possible upgrade would be to check N times and then to give up. But for now it looks good. :top: